src/MainWindow.vala

```diff
@@ -93,6 +93,13 @@ public class Camera.MainWindow : Gtk.Window {
 
             return 0;
         });
+
+        this.configure_event.connect ((event) => {
```

Comment:
I think you don't need `this` here.

Reply:
How should window resize be triggered?

Reply:
I mean that you could just write `configure_event.connect`, and Vala will understand you are talking about the signal, not some local variable, as it does for properties above.

Reply:
ah... I see! You are right!

Reply:
@BaptisteGelez other code lines have also a `this.`. So it looks better to leave it here in this case. (just code style)

Reply:
@BaptisteGelez could you remove your `block` from this pr please?